src/app/collective-rewards/rewards/backers/BackerRewardsTable.tsx

```diff
@@ -37,6 +37,8 @@ const tableHeaders: TableHeader[] = [
     label: 'Total Allocations',
     className: 'w-[18%]',
     sortKey: RewardsColumnKeyEnum.totalAllocationPercentage,
+    // eslint-disable-next-line quotes
+    tooltip: "Your allocations as a share % of the total Backer's allocations for the Builder",
```

Comment:
suggested simplified alternative: Your percentage of the Builder total allocations